CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -23,16 +23,16 @@ enum ItemType {
     }
 }
 
-enum SectionType {
+enum SectionType: CaseIterable {
     case yours
```

Comment:
I'm not going to look thru the whole PR for this, but adding on to Lucy's early comment, let's keep everything alphabetized